<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QMainWindow" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDialog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginInformation">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <widget class="QLabel" name="rightLogoLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>17</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource/resource.qrc">:/svpk_lib/images/sovajo_logo/logo_cn_thin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftLogoLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>86</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="resource/resource.qrc">:/svpk_lib/images/sovajo_logo/lgo_thin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">outline: none;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>205</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUserName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSystem_Information">
   <property name="text">
    <string>System Information</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>loginBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
